Dockerfile

```diff
@@ -92,7 +92,7 @@ COPY --chown=node:node package-lock.json package-lock.json
 
 USER node
 
-RUN npm install
+RUN npm ci --ignore-scripts
```

Comment:
We found in Make that Cypress needs one post install script to get its binary, but you can add it manually here:

`./node_modules/.bin/cypress install`